lib/TPP/TileConsumerAndFuseProducers.cpp

```diff
@@ -650,6 +682,12 @@ struct TileConsumerAndFuseProducers
       }
     }
 
+    // Pattern to replace iter args of the outer most loop with region args of
+    // the inner most one.
+    RewritePatternSet patterns1(&getContext());
```

Comment:
no silly variable names please.

why can't this be added in the pattern below?